verl/workers/fsdp_workers.py

```diff
@@ -554,14 +554,17 @@ def save_checkpoint(self, local_path, hdfs_path=None, global_step=0):
                                      device_id=torch.cuda.current_device(),
                                      load_grad=self._is_offload_grad)
 
-        self.checkpoint_manager.save_checkpoint(local_path=local_path, hdfs_path=hdfs_path, global_step=global_step)
+        self.checkpoint_manager.save_checkpoint(local_path=local_path,
+                                                hdfs_path=hdfs_path,
+                                                global_step=global_step,
+                                                remove_previous_ckpt=self.config.trainer.remove_previous_ckpt_in_save)
```

Comment:
You cannot use `self.config.trainer.remove_previous_ckpt_in_save` here.
Instead, you should add a new param `remove_previous_ckpt` in L548 and use it here.

The `self.config.trainer.remove_previous_ckpt_in_save` will only be passed in the ray_trainer.py